### ISSUE TITLE
about TTA

### ISSUE THREAD

**Issue by LinPeiMing** [FVL2020/SC-NAFSSR#1]
The code 
 use_hflip: true
 use_vflip: true
does that mean you use TTA method?

**Comment by qiuzidian** [comment]
This represents the method of data augmentation used in the training dataset ^_^

**Comment by LinPeiMing** [comment]
so when you test, you don't use TTA, is that right?

**Comment by qiuzidian** [comment]
yes

**Comment by LinPeiMing** [comment]
loss += 0.0025 * (loss_photo + 0.1 * loss_smooth + loss_cycle)
What does this 0.0025 mean? I don't seem to read it in the paper. Is it used to supervise SCAM in equation 4? Then why is 0.0025?
There is also a small problem. The code: "num_blks = 32, width = 32", but " num_blks = 128 width = 128"is commented ? But the paper says that the NAFSSR-L is used? Which one is used?
